### DISCUSSION TITLE
Internal links not clickable on iOS

### DISCUSSION THREAD

**Discussion by RomanIakovlev** [requarks/wiki#3966] [Error / Bug Report]
I'm using wiki.js version 2.5.201. The problem is that the internal links (that is, links to the other wiki pages) are not clickable on the iOS. When I do a normal short tap on that link, nothing happens. When I do a long tap, it opens a page preview and I can open the link in a new tab.

I believe this behavior is a bug. If maintainers agree it's a bug, I'd be happy to try and fix it, especially if provided some minimal guidance.

**Comment by skledarg** [comment]
I tried reproducing this on Wiki.js 2.5.201 / iOS 14.5.1 in Safari and Brave browser and internal links work as expected. 

Do you use markdown or html on that page? Maybe you have the link set to open in a new window and the browser might be blocking the popup? Can you post the source html of the link that isn't working for you?

**Reply by RomanIakovlev** [reply]
One important addition to my original post, sorry I forgot to mention it before. The link that I had was on the same line with the checkbox / todo. Here's the html:
```
<ul class="todo-list">
  <li>
    <a href="/test-page" class="is-internal-link is-valid-page">
      <label class="todo-list__label">
        <input disabled="disabled" type="checkbox">
          <span class="todo-list__label__description">/test-page</span>
      </label>
    </a>
  </li>
</ul>
```
I can confirm normal links, outside of todo/checkbox, work fine on iOS, it's only about these todo-links. I created the page using the WYSIWYG editor.
